internal/pkg/keeper/api/kv.go

```diff
@@ -0,0 +1,120 @@
+//
+// Copyright (C) 2024 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package api
+
+import (
+	"errors"
+	"net/http"
+	"net/url"
+	"path"
+
+	"github.com/edgexfoundry/go-mod-configuration/v3/internal/pkg/keeper/dtos"
+	httpUtils "github.com/edgexfoundry/go-mod-configuration/v3/internal/pkg/keeper/utils/http"
+)
+
+// KV is used to manipulate the K/V API
+type KV struct {
+	c *Caller
+}
+
+// KV is used to return a handle to the K/V apis
+func (c *Caller) KV() *KV {
+	return &KV{c}
+}
+
+// Get is used to lookup a single key. The returned pointer
+// to the KVPair will be nil if the key does not exist.
+func (k *KV) Get(key string) (res dtos.MultiKVResponse, err error) {
+	pathParams := url.Values{}
+	pathParams.Add(Plaintext, "true")
+
+	url := path.Join(ApiKVRoute, key)
+	errResp, err := httpUtils.GetRequest(&res, k.c.baseUrl, url, pathParams)
+	if err != nil {
+		return res, err
+	}
+	if errResp.StatusCode != 0 {
+		return res, errors.New(errResp.Message)
+	}
+	return res, nil
+}
+
+func (k *KV) Keys(key string) (res dtos.MultiKeyResponse, err error) {
```

Comment:
missing comment

Reply:
File removed, using the http clients from core-contract.